libvmaf/meson_options.txt

```diff
@@ -0,0 +1,9 @@
+option('tests', 
+    type: 'boolean', 
+    value: true, 
+    description: 'Enable libvmaf tests')
+
+option('documentation', 
+    type: 'boolean', 
+    value: true, 
+    description: 'Build documentation')
```

Comment:
Can you add a newline at the end of this file?

Reply:
Done.

Reply:
Thanks